### DISCUSSION TITLE
Word Count?

### DISCUSSION THREAD

**Discussion by domanchi** [outline/outline#3936] [Ideas]
What are your thoughts on having a built-in word count functionality? For mid-sized to long documents, it's useful to know how much content there is in a measurable manner.

Some thoughts on design:

- It could emulate Google Docs, and have the feature be visible within a menu item (for Google Docs, it's under Tools > Word Count)
- It could be a consistent widget (like, near the "Keyboard Shortcuts" button on the bottom right) that shows the count of highlighted words (if applicable), over all words in the document

Thoughts?

**Comment by tommoor** [comment]
I don't see any harm in including this!

Probably best to have it behind a menu item though as continuously counting words would be a drain on performance.
